docker-for-windows/index.md

```diff
@@ -12,32 +12,29 @@ redirect_from:
 - /docker-for-windows/index/
 title: Get started with Docker for Windows
 ---
-Welcome to Docker for Windows!
 
-Docker is a full development platform for creating containerized apps, and
-Docker for Windows is the best way to get started with Docker on Windows
-systems.
+Welcome to Docker for Windows! Docker is a full development platform for
+creating containerized apps, and Docker for Windows is the best way to get
+started with Docker _on Windows_ systems.
 
-> **Got Docker for Windows?** If you have not yet installed Docker for Windows, see [Install Docker for Windows](install.md) for an explanation of stable
-and edge channels, system requirements, and download/install information.
-
->**Looking for system requirements?** Check out
-[What to know before you install](install.md#what-to-know-before-you-install), which has moved to the new install topic.
-{: id="what-to-know-before-you-install" }
+> See [Install Docker for Windows](install.md){: target="_blank" class="_"} for information on system requirements and stable & edge channels.
 
 ## Check versions
 
-Start your favorite shell (`cmd.exe`, PowerShell, or other) to check your versions of `docker` and `docker-compose`, and verify the installation.
+Start your favorite shell (`cmd.exe`, PowerShell, or other) and ensure your
+versions of `docker`, `docker-compose`, and `docker-machine` are up-to-date and
+compatible with `Docker.app`. Your output may differ if you are running
+different versions.
 
 ```none
-PS C:\Users\Docker> docker --version
-Docker version 17.03.0-ce, build 60ccb22
+PS C:\Users\Gordon> docker --version
+Docker version 17.12.0-ce, build c97c6d6
 
-PS C:\Users\Docker> docker-compose --version
-docker-compose version 1.11.2, build dfed245
+PS C:\Users\Gordon> docker-compose --version
```

Comment:
I would argue that it is not important to show the version of all the components.  I would leave only `docker version` or `docker --version`.

Reply:
less is more. agree.